common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -3,6 +3,22 @@
 require "spec_helper"
 require "dependabot/clients/azure"
 
+RSpec.shared_examples "Successfully completes get request with auth headers" do |credential|
+  before do
+    stub_request(:get, base_url).
+      with(headers: credential["headers"]).
+      to_return(status: 200, body: '{"result": "Success"}')
+  end
+
+
+  it "Using #{credential["token_type"]} token in credentials" do
+
```

Comment:
I think our linter is failing on this, if we get rid of the blank line it should pass 👍